nservicebus/azure-storage-persistence/azure-storage-performance-tuning.include.md

```diff
@@ -0,0 +1,38 @@
+Azure Storage Persistence and Transport are network IO intensive. Every operation performed against the storage implies one or more network hops, most of which are small http requests to a single IP address (of the storage cluster). By default the .NET framework has been configured to be very restrictive when it comes to this kind of communication:
+- It only allows 2 simultaneous connections to a single IP address by default
+- It doesn't trust the remote servers by default, so it verifies for revoked certificates on every request
+- The algorithms were optimized for larger payload exchanges, not for small requests
+
+	
+## Network Optimizations
+
+Performance can be drastically improved by overriding the following settings. The `ServicePointManager` class can be used for this purpose by changing its settings. The changes must be done before the application makes any outbound connection, ideally very early in the application's startup routine:
```

Comment:
> Performance can be drastically improved by overriding the following settings. The `ServicePointManager` class can be used for this purpose by changing its settings. The changes must be done before the application makes any outbound connection, ideally very early in the application's startup routine:

Wouldn't it read better as:

The performance can be improved by overriding the settings exposed by the `ServicePointManager` class:

NOTE: Settings changes must be applied before the application makes any outbound connection, ideally very early in the application's startup routine.